reactor-aeron-core/src/main/java/reactor/aeron/DefaultAeronInbound.java

```diff
@@ -146,4 +154,27 @@ public void subscribe(CoreSubscriber<? super DirectBuffer> destinationSubscriber
       }
     }
   }
+
+  private static class CanceledSubscriber implements CoreSubscriber<DirectBuffer> {
+
+    @Override
+    public void onSubscribe(Subscription s) {
+      // no-op
+    }
+
+    @Override
+    public void onNext(DirectBuffer directBuffer) {
+      LOGGER.debug("was received buffer in the already canceled subscriber, {}", directBuffer);
```

Comment:
I think this must be .warn()  "Received message, which will be dropped immediately due cancelled aeron inbound"